extensions/sovity-messenger/README.md

```diff
@@ -0,0 +1,42 @@
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://github.com/sovity/edc-extensions">
+    <img src="https://raw.githubusercontent.com/sovity/edc-ui/main/src/assets/images/sovity_logo.svg" alt="Logo" width="300">
+  </a>
+
+<h3 align="center">EDC-Connector Extension:<br />Sovity Messenger</h3>
+
+  <p align="center">
+    <a href="https://github.com/sovity/edc-extensions/issues/new?template=bug_report.md">Report Bug</a>
+    ·
+    <a href="https://github.com/sovity/edc-extensions/issues/new?template=feature_request.md">Request Feature</a>
+  </p>
+</div>
+
+## About this Extension
+
+This extension provides a convenient way to exchange messages between connectors.
+
+## Why does this extension exist?
+
+To provide a simple way to exchange messages between EDCs without complying with the JsonLd conventions and with minimal setup.
+
+## Demo
```

Comment:
Needs a bit of architecture / design diagrams

Reply:
I think we don't necessarily do.

The API surface of the SovityMessenger is much easier and almost explains itself

Reply:
Added